tests/Doctrine/Tests/ORM/Mapping/AbstractMappingDriverTest.php

```diff
@@ -722,7 +722,7 @@ public function testSqlResultSetMapping(): void
         self::assertEquals(['name' => 'id', 'column' => 'id'], $mapping['entities'][0]['fields'][0]);
         self::assertEquals(['name' => 'name', 'column' => 'name'], $mapping['entities'][0]['fields'][1]);
         self::assertEquals(['name' => 'status', 'column' => 'status'], $mapping['entities'][0]['fields'][2]);
-        self::assertEquals(['name' => 'phonenumbers.phonenumber', 'column' => 'number'], $mapping['entities'][0]['fields'][3]);
+        self::assertEquals(['name' => 'phonenumbers.phonenumber', 'column' => 'phone'], $mapping['entities'][0]['fields'][3]);
```

Comment:
It's embarrassing that I had to rename a column from a reserved word to pass the test. Most likely I will return and think about how to solve it more correctly

Reply:
It's a known issue: https://github.com/doctrine/dbal/issues/4357.

Reply:
Revert use reserved words